enctests/testframework/main.py

```diff
@@ -225,27 +214,28 @@ def create_source_config_files(args):
         f'Make sure to do adjustments of in point, duration and so on.'
     )
 
-
-def get_configs(args, root_path, config_type):
-    configs = []
+def get_source_configs(args, root_path, config_type):
+    sourceconfigs = []
     for item in scantree(args, root_path, suffix=config_type):
         path = Path(item.path)
         if path.suffix == config_type:
-            configs.extend(parse_config_file(path))
+            try:
+                sourceconfigs.append(SourceConfig(path))
+            except Exception as e:
+                print(f"ERROR: Failed to load source config {path} error is: {e}")
 
-    return configs
+    return sourceconfigs
 
 
-def tests_only(test_configs):
-    """
-    Scan the test-configs for just the test configurations (since it can be co-mingled with output and source info).
-    Each test must start with the label test" to not confuse it with other configs.
-    """
+def get_configs(args, root_path, config_type):
```

Comment:
We could consider calling this "get_test_configs" ?

Reply:
Makes sense to make, made a patch for it.